### PR TITLE
fix:updated css to align checkbox and label to center in checkbox-widget

### PR DESCRIPTION
**Pull request by abhishek-bandameedi**
### Description:

Updated the alignment of checkbox and its label to center when the size of label is increased.

##### Before:
image

##### After:
image

## Summary by CodeRabbit

- **New Features**
	- Introduced end-to-end tests for the Checkbox widget to validate functionality and visual alignment.
	- Added unit tests for the CheckboxComponent to ensure correct rendering and style application.

- **Improvements**
	- Enhanced styling of the Checkbox widget for better visual presentation and alignment within the user interface.

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Checkbox/CheckBox2_spec.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    it("Checkbox and it's label should be aligned in center", function () {
+      cy.openPropertyPane("checkboxwidget");
+      cy.get(".sc-hINMOq").click({ force: true });
+      cy.get("#radix-61-trigger-style > .sc-bcXHqe").click({ force: true });
+
+      _.propPane.EnterJSContext("Font size", "");
+      _.propPane.EnterJSContext("Font size", "4rem");
+
+      cy.get(".bp3-control").should("have.css", "align-items", "center");
+    });
```
[the remark this answers is not in the document]
Comment (abhishek-bandameedi) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Checkbox/CheckBox2_spec.js]: addressed the suggested comment
1f26c45368456239ad87c0402f0ba39511b96621

**Comment by sagar-qa007** [comment]
@abhishek-bandameedi Can you please explain why we need cypress here? We have seen there is change in src with style updation. This logic can be tested with Jest unit tests or React testing library. Cypress is redundant here.

**Comment by abhishek-bandameedi** [comment]
> @abhishek-bandameedi Can you please explain why we need cypress here? We have seen there is change in src with style updation. This logic can be tested with Jest unit tests or React testing library. Cypress is redundant here.

Updated unit tests
2e8126e2508a9f27b3a690b8f89c512a5f052f5f

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Checkbox/CheckBox2_spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import * as _ from "../../../../../support/Objects/ObjectsCore";
+
+describe(
+  "Checkbox Widget Functionality",
+  { tags: ["@tag.Widget", "@tag.Checkbox"] },
+  function () {
+    before(() => {
+      _.agHelper.AddDsl("emptyDSL");
+    });
+    it("Add new widget", () => {
```
Comment (sagar-qa007) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Checkbox/CheckBox2_spec.js:10]: Could you please help me understand the necessity of Cypress test cases in this context? I believe unit testing might be sufficient in this scenario.
Your insights would be appreciated.

**Pull request closed without merging**